Content.Server/Kitchen/EntitySystems/ReagentGrinderSystem.cs

```diff
@@ -157,7 +157,7 @@ private void OnContainerModified(EntityUid uid, ReagentGrinderComponent reagentG
             var outputContainer = _itemSlotsSystem.GetItemOrNull(uid, SharedReagentGrinder.BeakerSlotId);
             _appearanceSystem.SetData(uid, ReagentGrinderVisualState.BeakerAttached, outputContainer.HasValue);
 
-            if (reagentGrinder.AutoMode != GrinderAutoMode.Off && !HasComp<ActiveReagentGrinderComponent>(uid))
+            if (reagentGrinder.AutoMode != GrinderAutoMode.Off && !HasComp<ActiveReagentGrinderComponent>(uid) && TryComp<ApcPowerReceiverComponent>(uid, out var powerReceiverComponent) && powerReceiverComponent.Powered)
```

Comment:
use _power.IsPowered

Reply:
Updated check to use IsPowered extension.